tests/lib/prepare.sh

```diff
@@ -950,7 +950,8 @@ EOF
 
     # download the core20 snap manually from the specified channel for UC20
     if os.query is-core20; then
-        snap download core20 --channel="$BASE_CHANNEL" --basename=core20
+        #snap download core20 --channel="$BASE_CHANNEL" --basename=core20
+        curl -o core20.snap https://people.canonical.com/~mvo/tmp/core20_20211129_amd64.snap
```

Comment:
was this intentional as part of this PR?

Reply:
Absolutely not, sorry for this!